src/routes/router.js

```diff
@@ -30,6 +30,9 @@ import configureStore from './middleware'
 
 import * as segment from 'servers/segment/index'
 
+const ORGANIZATION_TAB_REGEX = /tab=(\w+)/
```

Comment:
I believe it's more correct to use ?tab=, rather than /tab, for what is effectively a parameter instead of a query (indicated by the = sign)